lib/hanami/utils/kernel.rb

```diff
@@ -25,6 +25,14 @@ module Kernel
       # @see Hanami::Utils::Kernel.Integer
       NUMERIC_MATCHER = /\A([\d\/\.\+iE]+|NaN|Infinity)\z/.freeze
 
+      # Default precision for bigdecimal with rational number
+      #
+      # @since x.x.x
+      # @api private
+      #
+      # @see Hanami::Utils::Kernel.BigDecimal
+      BIGDECIMAL_DEFAULT_PRECISION = 1
```

Comment:
you are right, when you use big decimal is for big precision. I don't know if a good value is 16 but I think is big accuracy  hehe, I'll change with that value